src/actions/admin-access-actions.js

```diff
@@ -35,9 +37,9 @@ export const ADMIN_ACCESS_ADDED          = 'ADMIN_ACCESS_ADDED';
 export const ADMIN_ACCESS_DELETED        = 'ADMIN_ACCESS_DELETED';
 
 
-export const getAdminAccesses = (term = null, page = 1, perPage = 10, order = 'id', orderDir = 1 ) => (dispatch, getState) => {
-    const { loggedUserState } = getState();
-    const { accessToken }     = loggedUserState;
+export const getAdminAccesses = (term = null, page = 1, perPage = 10, order = 'id', orderDir = 1 ) => async (dispatch, getState) => {
+
+    const accessToken = await getAccessToken();
```

Comment:
@santipalenque something that i didnt put to much effort on the former POC ( PKCE support)  was error handling
truth is that async method "getAccessToken" could throw an exception 
so i was thinking ( open to suggestions here ) on a way to handling this errors gracefully
one thing that came to my mind is this

 ```
let accessToken = null;
    try {
        accessToken = await getAccessToken();
    }
    catch(e){
        console.error(e);
        Swal.fire("ERROR", e.toString(), "error");
        initLogOut();
        return Promise.reject(e);
    }
```

let me know what u think :)
regards